documentation/automating_windows_targets.md

```diff
@@ -1,6 +1,6 @@
 # Automating Windows targets
 
-Examples of common Bolt usage on Windows
+Here are some common tasks that you can accomplish with Bolt on Windows targets.
```

Comment:
```suggestion
Here are some common use cases that you can accomplish with Bolt on Windows targets.
```
'Tasks' just feels like kind of a loaded word, but there's not a lot of great synonyms :P

Reply:
I agree. I could not for the life of me think of a synonym. Use cases is perfect